packages/perseus/src/widgets/interactive-graphs/graphs/linear-system.tsx

```diff
@@ -28,12 +34,70 @@ const LinearSystemGraph = (props: LinearSystemGraphProps) => {
     const {dispatch} = props;
     const {coords: lines} = props.graphState;
 
+    const {strings, locale} = usePerseusI18n();
+    const id = React.useId();
+
+    const linesAriaInfo = lines.map((line, i) => {
+        return {
+            pointsDescriptionId: id + `-line${i + 1}-points`,
+            interceptDescriptionId: id + `-line${i + 1}-intercept`,
+            slopeDescriptionId: id + `-line${i + 1}-slope`,
```

Comment:
Minor: as long as we're using a template string, we could include `id` in it too:

```suggestion
            pointsDescriptionId: `${id}-line${i + 1}-points`,
            interceptDescriptionId: `${id}-line${i + 1}-intercept`,
            slopeDescriptionId: `${id}-line${i + 1}-slope`,
```